streaming/base/format/base/writer.py

```diff
@@ -390,15 +370,17 @@ def __init__(self,
                  keep_local: bool = False,
                  compression: Optional[str] = None,
                  hashes: Optional[List[str]] = None,
-                 size_limit: Optional[int] = 1 << 26) -> None:
+                 size_limit: Optional[int] = 1 << 26,
+                 **kwargs: Any) -> None:
```

Comment:
Dict[str, Any]

Reply:
See my above response for the same comment.